Userland/Libraries/LibJS/Runtime/AsyncFunctionDriverWrapper.cpp

```diff
@@ -15,53 +15,115 @@ namespace JS {
 
 ThrowCompletionOr<Value> AsyncFunctionDriverWrapper::create(Realm& realm, GeneratorObject* generator_object)
 {
-    auto wrapper = MUST_OR_THROW_OOM(realm.heap().allocate<AsyncFunctionDriverWrapper>(realm, realm, generator_object));
```

Comment:
commit description typo: `handeling` -> `handling`, `propperly` -> `properly`.
Also please do a little grammar check on the commit descriptions.